### ISSUE TITLE
[Development|Keyboard] HLR: Fix post-wizard focus management

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#15150]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## User Story or Problem Statement

When a user completes the Higher-Level Review (HLR) wizard, the introduction page content becomes visible but focus is lost.

Focus needs to be set on the `H1` and the page scrolled to the top at this point.

## Details

- See this Slack discussion: https://dsva.slack.com/archives/C0113MPTGH5/p1601923906011400
- Once a content review has been done, the text of the `H1` may change, but this will be applied in future work/new issue.

## Acceptance Criteria

- [ ] Focus moved to `H1` on wizard completion
- [ ] Page scrolled to top

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

All

## Steps to Recreate

1. No need to log in
1. Go to https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/introduction
1. Choose "Disability compensation claim" when asked for type of claim
1. Choose "No" for legacy appeals question
1. Scroll down the to the bottom of the page
1. Use "Request a Higher-Level Review" button
1. Note the introduction page content is now visible, but the focus is lost and the scroll position has not moved

## Solution (if known)

- Move focus to `H1`
- Scroll to the top of the page
